<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewindow</class>
 <widget class="QMainWindow" name="gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gamewindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kd :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kp :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jogar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="KpSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="KdSpinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>gSizeW :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="gSizeWSpinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>deW :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KrW :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>k_larg :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="deWSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="krWSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kLargSpinBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>defenderLine :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="defenderLineSpinBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Strategy :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="StrategycomboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FIXED</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FULL_ATK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SAFE_FULL_ATK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FUZZY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KNN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IMPOSTOR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DOWNBELOW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FUZZYLEIRO</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
